securedrop/source_templates/logout.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 {% block body %}
 <nav>
-  <a href="{{ url_for('main.login') }}" class="btn" aria-label="{{ gettext('Log In') }}">
+  <a href="{{ url_for('main.login') }}" class="btn" aria-label="{{ gettext('Log In Again Using A Previous Codename') }}">
```

Comment:
Could this be simplified to either of:

```suggestion
  <a href="{{ url_for('main.login') }}" class="btn" aria-label="{{ gettext('Log In Again') }}">
```

"Again" makes sense here!  But at that point a source already knows what's involved.  Or:

```suggestion
  <a href="{{ url_for('main.login') }}" class="btn" aria-label="{{ gettext('Log Back In' }}">
```

Reply:
I think the suggestion of "Log Back In" is clearer here, so I propose we go with that.